PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,4 +16,6 @@
 - [ ] I have performed a self-review of my own code
 - [ ] If my code involves visual changes, I am adding applicable screenshots to this thread
 
+<!--
 PS: [Read how to write the perfect pull request](https://blog.github.com/2015-01-21-how-to-write-the-perfect-pull-request/)
```

Comment:
Nitpick: markdown link syntax still present, though it will never render as a link because it's in a comment